lib/harvest-notifier/harvest.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "httparty"
+
+module HarvestNotifier
+  class Harvest
+    include HTTParty
+
+    base_uri "https://api.harvestapp.com/api/v2"
+    logger ::Logger.new STDOUT
+
+    def initialize(token, account_id)
+      headers = { "Authorization" => "Bearer #{token}", "Harvest-Account-Id" => account_id }
+      self.class.headers headers
+    end
+
+    def users_list
+      self.class.get("/users.json?is_active=true")
+    end
+
+    def time_report_list(from, to = from)
+      query = "?from=#{prepare_date(from)}&to=#{prepare_date(to)}"
```

Comment:
Could we try to use `Hash#to_query` (from [ActiveSupport](https://apidock.com/rails/Hash/to_query)) to store this like a hash?

```ruby
params = {from: prepare_date(from), to: prepare_date(to)}
self.class.get("/reports/time/team.json?#{params.to_query}")
```